permits/views.py

```diff
@@ -117,7 +118,26 @@ def progress_bar_context(request, permit_request, current_step_type):
     return {"steps": steps, "previous_step": previous_step}
 
 
+def check_mandatory_2FA(
+    view=None, redirect_field_name="next", login_url="profile", if_configured=False
```

Comment:
`is_configured` au lieu de `if_configured` non ?

Reply:
It is ok, adopted from the original **otp_required** method, it means If ``True``, an authenticated user with no confirmed
 OTP devices will be allowed. Default is ``False``